Plugin/BypassWebApiTwoFactorAuth.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * This file is part of the MarkShust DisableTwoFactorAuth package
+ *
+ * @author Jitheesh V O <jitheeshvo@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code
+ */
+declare(strict_types=1);
+
+namespace MarkShust\DisableTwoFactorAuth\Plugin;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Integration\Api\AdminTokenServiceInterface;
+use Magento\TwoFactorAuth\Model\AdminAccessTokenService;
+
+/**
+ * Class BypassWebApiTwoFactorAuth
+ *
+ * @package MarkShust\DisableTwoFactorAuth\Plugin
+ */
+class BypassWebApiTwoFactorAuth
+{
+    const XML_PATH_CONFIG_API_ENABLE = 'twofactorauth/general/enable_api';
+
+    /** @var ScopeConfigInterface */
+    private $scopeConfig;
+    /**
+     * @var AdminTokenServiceInterface
+     */
+    private AdminTokenServiceInterface $adminTokenService;
+
+    public function __construct(
+        AdminTokenServiceInterface $adminTokenService,
+        ScopeConfigInterface $scopeConfig
+    ) {
+        $this->scopeConfig = $scopeConfig;
+        $this->adminTokenService = $adminTokenService;
+    }
+
+    /**
+     * This will bypass 2fa and allow us to use existing token generate end-point
+     * Recommended to use this until third-party service is ready to configure 2fa
+     *
+     * @param AdminAccessTokenService $subject
+     * @param \Closure                $proceed
+     * @param                         $username
+     * @param                         $password
+     *
+     * @return string
+     * @throws \Magento\Framework\Exception\AuthenticationException
+     * @throws \Magento\Framework\Exception\InputException
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function aroundCreateAdminAccessToken(
+        AdminAccessTokenService $subject,
+        \Closure $proceed,
```

Comment:
Another one of my pedantic reviews (I apologize and desperately need a linting GitHub action or something setup), but the `\Closure` is considered a fully qualified class name and should be hoisted to the top of this file with `use \Closure;`. Related docblock should also be updated.